styles/globals.scss

```diff
@@ -1,8 +1,12 @@
 @import 'bootstrap/dist/css/bootstrap.min.css';
 
```

Comment:
it looks like you have an autoformatter on that I don't? we should sync up on that